### ISSUE TITLE
Add Batch (.bat) Syntax Highlighting

### ISSUE THREAD

**Issue by jbonadiman** [silverbulletmd/silverbullet#885]
Please could batch (.bat)/cmd syntax highlighting be added to codeblocks?

**Reproduce Issue**
  1. Insert code block and after the first ``` enter either bat, batch or cmd
  2. No syntax highlighting provided

Thanks in advance!

**Comment by MrMugame** [comment]
I couldn't find any good codemirror mode for batch files, so this is probably something that has to wait until there is a proper plugin api for language modes.

**Comment by zefhemel** [comment]
Indeed, I would have hoped it's available from here: https://github.com/codemirror/legacy-modes/tree/main/mode but I don't see it. So indeed somebody would have to create one first. Unless it's compatible with powershell, but I don't think so?
